build/buildkite/script-evaluation-test.yml

```diff
@@ -0,0 +1,17 @@
+# A nightly job which downloads script evaluation dumps from S3 and runs a regression test.
+steps:
+  - label: "Download and unzip dump files"
+    command: |
+      nix-shell --run 'LOCAL_DIR=$(realpath .)/../mainnet-script-dump-downloaded ./scripts/s3-sync-unzip.sh s3://plutus/mainnet-script-dump/ \\*.event.bz2'
```

Comment:
Why isn't just passing the relative path enough? This maybe needs some comments about where the dump is on the machine - why is it at that path?

If you could just pass the relative path then you could also pass it with 
```
env:
  LOCAL_DIR: "../mainnet-script-dump-download"
```
which would make the pipeline a bit simpler.

Reply:
Passing relative path should be enough - just want to guard against accidental change to the working directory in the scripts.

Added a comment about where the dump is on the machine:

```
# The "../" is the parent of the pipeline-specific directory. This makes the script dump easily accessible
# regardless of which buildkite pipeline we use.
```

Reply:
It is, but good point about $HOME. I'll make a note here.

Reply:
Yeah, I think $HOME is better. I'll update it.